demo.yml

```diff
@@ -27,12 +27,12 @@ services:
     build:
       context: ./ember
       args:
-        EMBER_GIT_REPO: "${EMBER_GIT_REPO}"
-        EMBER_GIT_BRANCH: "${EMBER_GIT_BRANCH}"
+        EMBER_GIT_REPO: "${EMBER_GIT_REPO:-}"
+        EMBER_GIT_BRANCH: "${EMBER_GIT_BRANCH:-}"
```

Comment:
`EMBER_GIT_REPO` and `EMBER_GIT_BRANCH` should be specified, else builds would not be possible. (Guess that means we need to add them to the .env file.

Other env vars have sensible defaults in the Ember app, if the var doesn't exist or has no value, so it's fine if we use the `${...:-}` notation

Reply:
What are the correct values for these for demo; and will those values always be the same for demo... or would they depend on what branch/commit was used ?

Reply:
* `EMBER_GIT_REPO=https://github.com/eclipse-pass/pass-ui`
* `EMBER_GIT_BRANCH=main`

These values are only used at build time, so they should work for all environments